pallets/pool-system/src/tranches.rs

```diff
@@ -2908,11 +2908,10 @@ pub mod test {
 		mod rebalance {
 			use super::*;
 
-			const TOTAL_ASSETS: Balance =
-				DEBT_RES
-					+ RESERVE_RES + DEBT_NONRES_1
-					+ RESERVE_NONRES_1
-					+ DEBT_NONRES_2 + RESERVE_NONRES_2;
+			const TOTAL_ASSETS: Balance = DEBT_RES
+				+ RESERVE_RES + DEBT_NONRES_1
+				+ RESERVE_NONRES_1
+				+ DEBT_NONRES_2 + RESERVE_NONRES_2;
```

Comment:
Before you get crazy with this and the `rustfmt`. Upgrade your current nightly rust tooling, because they have made a major version in the `rustfmt` without upgrading the version number 🥲